main.go

```diff
@@ -29,7 +29,8 @@ import (
 var (
 	sqlType          = goopt.String([]string{"--sqltype"}, "mysql", "sql database type such as [ mysql, mssql, postgres, sqlite, etc. ]")
 	sqlConnStr       = goopt.String([]string{"-c", "--connstr"}, "nil", "database connection string")
-	sqlDatabase      = goopt.String([]string{"-d", "--database"}, "nil", "Database to for connection. For Postgres, it's the table's schema name.")
+	sqlDatabase      = goopt.String([]string{"-d", "--database"}, "nil", "Database to for connection. For Postgres, it's the table's schema name. Overrides schemas if also provided")
+	sqlSchemas       = goopt.String([]string{"-s", "--schemas"}, "nil", "If postgres, allows for multiple postgres schemas")
```

Comment:
Do we need to add this flag? sqlDatabase is treated as postgres table schema

Reply:
We could for sure just reuse the same flag too. The idea was to somehow make it a bit more clear when using it.
Any preference?

Reply:
We need to be clear that for postgres`sqlSchema` is replacing `sqlDatabase` if `sqlSchema` is specified. The rest of the code change LGTM.

Can you also update the README?